### ISSUE TITLE
problems with grails-2.3.RC1.

### ISSUE THREAD

**Issue by ghost** [joergrech/KickstartWithBootstrap#44]
I create a new app with grails-2.3.RC1 then install kickstartWithBootstrap.
Create a new domain class, a new controller and then run generate-views mypackage.Mydomain.
The error:

Error Error running script generate-views supplierregister.Sector : groovy.lang.MissingMethodException: No signature of method: static org.codehaus.groovy.grails.orm.hibernate.cfg.GrailsDomainBinder.getMapping() is applicable for argument types: (org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass) values: [Artefact > Sector]
Possible solutions: getMapping(java.lang.Class), getMapping(org.codehaus.groovy.grails.commons.GrailsDomainClass) (Use --stacktrace to see the full trace)

I have no problems without kickstartWithBootstrap.

**Comment by assiomatica** [comment]
you can fix it in templates _forms.gsp

line 7  
change so:

```
if (hasHibernate && (new org.codehaus.groovy.grails.orm.hibernate.cfg.GrailsDomainBinder().getMapping(domainClass))?.identity?.generator == 'assigned') {
```

thanks to 
https://github.com/burtbeckwith/grails-filterpane/commit/e82eb1fbd3f8b274e415ca254f970d547aa468ed

**Comment by joergrech** [comment]
I just released version 1.1.0 that works with Grails 2.3
